utkdigitalinitiatives/utk_isl_xml_forms
# Trac 929

**JIRA Ticket TRAC-929**: https://jira.lib.utk.edu/browse/TRAC-929

# What does this Pull Request do?

Rachel Caldwell requested specific changes to the form for submitting a Graduate Thesis/Dissertation. (see TRAC-926)

Rachel specified changes to headers and to directional text as follows:
- ADVISOR/MAJOR PROFESSOR FIRST NAME/GIVEN NAME
-- Enter first name and, if used, middle name or initial. Do not include honorifics or titles such as “Dr.” or “PhD.” Examples: Stephen G. or Ruth Bader.

- ADVISOR/MAJOR PROFESSOR LAST NAME/FAMILY NAME
-- Enter family name or last name. Examples: Breyer or Ginsburg.

- ADVISOR/MAJOR PROFESSOR SUFFIX
-- Enter suffix. Example: Jr. 

- COMMITTEE MEMBER FIRST NAME/GIVEN NAME
-- Enter first name and, if used, middle name or initial. Do not include honorifics or titles such as “Dr.” or “PhD.” Examples: Stephen G. or Ruth Bader.

- COMMITTEE MEMBER LAST NAME/FAMILY NAME
-- Enter family name or last name. Examples: Breyer or Ginsburg.

- COMMITTEE MEMBER SUFFIX
-- Enter suffix. Example: Jr. 

# What has changed?
The only file that was changed is:
utkdigitalinitiatives/utk_isl_xml_forms(TRAC-929)/xml_forms/UTK_ir_etds.xml

The revised form should reflect these change requests.
This change does not affect documentation, code dependencies, or any other files.

# How should this be tested?
In vagrant bring up TRACE(your-favorite-test-branch)
- Log in as admin.
- In the Dashboard, go to Islandora >>Form Builder
- Click on Import Form
- Name the form TRAC-929
- Select the file utkdigitalinitiatives/utk_isl_xml_forms(TRAC-929)/xml_forms/UTK_ir_etds.xml
from your desktop github repository.
- Once the form TRAC-929 has been created, it will appear on the list on the Form Builder table.
- Click on Associate for TRAC-929 and fill out the form as shown below.
![0008_define_the_form_association](https://user-images.githubusercontent.com/11573234/32957309-5510a796-cb89-11e7-916c-f748c1a383af.JPG)
- Log out.
- Log in as userA
- Go to Sumit, select Submit Graduate Thesis or Dissertation
- The next page will say Select a Form.  Select TRAC-929
- This will bring you to the submission form.
- Scroll down to see if Rachel's changes are in place as shown below.
![0012_form_display_with_rachels_changes](https://user-images.githubusercontent.com/11573234/32957529-ec109b56-cb89-11e7-86f9-cb8b32baa58c.JPG)


# Interested parties
@CanOfBees @markpbaggett 
Tag (@ mention) interested parties.
